bokehjs/gulp/tasks/generate_defaults.py

```diff
@@ -104,8 +104,5 @@ def output_defaults_module(filename, defaults):
 
 output_defaults_module(filename = os.path.join(dest_dir, 'common/defaults.coffee'),
                        defaults = all_json)
-output_defaults_module(filename = os.path.join(dest_dir, 'widget/defaults.coffee'),
+output_defaults_module(filename = os.path.join(dest_dir, 'models/widgets/defaults.coffee'),
```

Comment:
> This would be confusing after compilation (whatever this refers to - name doesn't suggest at all), because it would generate bokeh-widgets.js and we already generate that.

Ah that's a good point.... I guess my main suggestion is that the "build" coffee files (that do nothing but collect `require` statements for a built  final module) get a consistent naming convention, and (possibly) a separate "place" for them all to go together.